src/Microsoft.Health.Fhir.Core/Features/TaskManagement/ITask.cs

```diff
@@ -0,0 +1,21 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Threading.Tasks;
+
+namespace Microsoft.Health.Fhir.Core.Features.TaskManagement
```

Comment:
I don't think there is anything FHIR specific to all these Task related classes. It seems to me that instead of them being in Fhir.Core, it should be a separate project. And that will also keep things modular so that if other users want to use a different task management mechanism they can easily insert their own implementation.

Reply:
Agree, moved to a separated project.